### PR TITLE
Improve sequence

### PR DESCRIPTION
**Pull request by vaskonov**

### CODE REVIEW

File: utils/partitions.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ exports.partition = function(dataset, testSetStart, testSetCount) {
  * @note code adapted from Heather Arthur:  https://github.com/harthur/classifier/blob/master/test/cross-validation/cross-validate.js
  */
 exports.partitions = function(dataset, numOfPartitions, callback) {
-	var shuffledDataset = _.shuffle(dataset);
+	// var shuffledDataset = _.shuffle(dataset);
```
Comment (erelsgl) [inline utils/partitions.js]: Why did you remove the shuffling? 

In cross-validation, it is common to shuffle the data in order to make sure that there is no bias in the partition.

Comment (vaskonov) [inline utils/partitions.js]: I used it for check
Now it's fixed
